zendriver/core/connection.py

```diff
@@ -296,6 +296,11 @@ def remove_handlers(
                 page.remove_handlers(cdp.network.RequestWillBeSent, handler)
         """
 
+        if handler and not event_type:
+            raise TypeError(
```

Comment:
super nit, but I think this should actually be `ValueError`

```suggestion
            raise ValueError(
```

https://docs.python.org/3/library/exceptions.html

> Passing arguments of the wrong type (e.g. passing a [list](https://docs.python.org/3/library/stdtypes.html#list) when an [int](https://docs.python.org/3/library/functions.html#int) is expected) should result in a [TypeError](https://docs.python.org/3/library/exceptions.html#TypeError), but passing arguments with the wrong value (e.g. a number outside expected boundaries) should result in a [ValueError](https://docs.python.org/3/library/exceptions.html#ValueError).

Reply:
(because `None` is an allowed type per the annotation, it's just an issue when both values are None)